Common/src/main/java/com/blamejared/crafttweaker/api/zencode/impl/CraftTweakerDefaultScriptRunConfiguration.java

```diff
@@ -0,0 +1,109 @@
+import org.openzen.zencode.java.module.converters.JavaNativeConverterBuilder;
+import org.openzen.zencode.shared.CompileException;
+import org.openzen.zenscript.parser.BracketExpressionParser;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.function.BiConsumer;
+import java.util.function.Consumer;
+
+public final class CraftTweakerDefaultScriptRunConfiguration {
+    
+    public static final ScriptLoadingOptions.ScriptRunConfiguration DEFAULT_CONFIGURATION = CraftTweakerDefaultScriptRunConfiguration::configure;
+    
+    private static void configure(
+            final BiConsumer<String, BracketExpressionParser> registrationFunction,
+            final Consumer<JavaNativeModule> addingFunction,
+            final ScriptingEngine engine
+    ) throws CompileException {
+        
+        registerModules(registrationFunction, addingFunction, engine);
+    }
+    
+    private static void registerModules(
+            final BiConsumer<String, BracketExpressionParser> registrationFunction,
+            final Consumer<JavaNativeModule> addingFunction,
+            final ScriptingEngine engine
+    ) throws CompileException {
+        
+        final List<JavaNativeModule> modules = new ArrayList<>();
+        final CrTJavaNativeConverterBuilder nativeConverterBuilder = new CrTJavaNativeConverterBuilder();
+        
+        //Register crafttweaker module first to assign deps
+        final JavaNativeModule crafttweakerModule =
+                addModule(registrationFunction, addingFunction, modules::add, engine, CraftTweakerConstants.MOD_ID, CraftTweakerConstants.MOD_ID, nativeConverterBuilder);
+        
+        final Set<String> rootPackages = new HashSet<>(CraftTweakerRegistry.getRootPackages());
+        rootPackages.remove(CraftTweakerConstants.MOD_ID);
+        for(String rootPackage : rootPackages) {
+            addModule(registrationFunction, addingFunction, modules::add, engine, rootPackage, rootPackage, nativeConverterBuilder, crafttweakerModule);
+        }
+        
+        addModule(
+                registrationFunction,
+                addingFunction,
+                it -> CraftTweakerRegistry.getExpansions()
+                        .values()
+                        .stream()
+                        .flatMap(List::stream)
+                        .forEach(it::addClass),
+                engine,
+                "expansions",
+                "",
+                nativeConverterBuilder,
+                modules.toArray(JavaNativeModule[]::new)
+        );
+        
+        nativeConverterBuilder.headerConverter.reinitializeAllLazyValues();
+    }
+    
+    private static JavaNativeModule addModule(
+            final BiConsumer<String, BracketExpressionParser> registrationFunction,
+            final Consumer<JavaNativeModule> addingFunction,
+            final Consumer<JavaNativeModule> applyFunction,
```

Comment:
NAMES.
Even more so for the types.

I am pretty sure that in a few months we will have no idea whatsoever what the difference between those two consumers would be when only looking at the method signature.